std/range/package.d

```diff
@@ -4355,15 +4357,357 @@ if (Ranges.length && allSatisfy!(isInputRange, Ranges))
     }
 }
```

Comment:
FYI:

```
std/range/package.d(4015:8)[warn]: Public declaration 'Zip' has no documented example.
```

This is due to https://github.com/dlang-community/D-Scanner/issues/566
I will have a look into fixing that one.

Reply:
Meh, it looks like we can't fix that one easily: https://github.com/dlang-community/D-Scanner/issues/566#issuecomment-375355609

Reply:
As a workaround I've moved the declaration of `ZipShortest` to later in the file so it won't confuse D-Scanner.